Edits to EPAS 13 - update clone schema setup steps #6336 (#6455)
EnterpriseDB/docs: product_docs/docs/epas/14/epas_guide/14_edb_clone_schema.mdx

-   `process_status_from_log`. This function displays the status of the cloning functions. The information is obtained from a log file you specify when invoking a cloning function. See [process_status_from_log](#process_status_from_log) for more information.
-   `remove_log_file_and_job`. This function deletes the log file created by a cloning function. You can also use this function to delete a job created by the non-blocking form of the function. See [remove_log_file_and_job](#remove_log_file_and_job) for more information.
-   `create_clone_log_dir`. This function creates a directory to store all the log files created by a cloning function.
-   `grant_clone_schema_privileges`. This function grants the privileges to clone the schema to a non-superuser.

> -   `revoke_clone_schema_privileges`. This function revokes the privileges to clone the schema from a non-superuser.

Author: ```suggestion
-   `revoke_clone_schema_privileges`. This function revokes the privileges to clone the schema from a non-superuser.

```

### REVOKE_CLONE_SCHEMA_PRIVILEGES

> You can revoke the clone schema privileges from a non-superuser using this function.

Author: ```suggestion
This function revokes the clone schema privileges from a non-superuser.
```

(1 row)
```

### Example: Clone a schema locally as a non-superuser

Create a non-superuser named `forcs`:

```sql
CREATE USER forcs PASSWORD `abc123`;
__OUTPUT__
CREATE ROLE
```

> Give CREATE privileges  on the `edb` database to `forcs`:

Author: ```suggestion
Give CREATE privileges on the `edb` database to `forcs`:
```
